pkg/kubelet/cri/remote/remote_runtime_test.go

```diff
@@ -47,7 +47,7 @@ func createAndStartFakeRemoteRuntime(t *testing.T) (*fakeremote.RemoteRuntime, s
 }
 
 func createRemoteRuntimeService(endpoint string, t *testing.T) internalapi.RuntimeService {
-	runtimeService, err := NewRemoteRuntimeService(endpoint, defaultConnectionTimeout)
+	runtimeService, err := NewRemoteRuntimeService(endpoint, defaultConnectionTimeout, nil)
```

Comment:
Needs NewNoopTracerProvider instead of nil.

Reply:
updated